torch/csrc/autograd/python_function.h

```diff
@@ -34,7 +34,6 @@ struct THPFunction {
 
     PyObject *needs_input_grad;
     PyObject *backward_hooks;
```

Comment:
No, no. The only solution would be to port `register_hook` to C (that's the only place where we use the attribute), but I think we can leave it as is for now if you want.